### ISSUE TITLE
Codegen for Android generates invalid syntax in CopyOfBuilder

### ISSUE THREAD

**Issue by rjuliano** [aws-amplify/amplify-cli#4864]
**Describe the bug**
If you have a model such as the `Blog` below, where there are no fields other than the id and connected fields.

```
type Blog @model
{
    id: ID!
    posts: [Post] @connection(name: "BlogPosts")
}
type Post @model
{
    id: ID!
    blog: Blog @connection(name: "BlogPosts")
    comments: [Comment!] @connection(name: "PostComments")
    text: String
}
type Comment @model
{
    id: ID!
    post: Post @connection(name: "PostComments")
}
```

The CLI generates the following for the Blog model:

```
public final class CopyOfBuilder extends Builder {
    private CopyOfBuilder(String id) {
      super.id(id);
      super;
    }
  }
```

I was able to workaround by adding a dummy field to Blog.

**Amplify CLI Version**
4.24.2

**To Reproduce**
Using a new or existing Amplify project setup for Android:
- Add the schema mentioned above.
- run `amplify codegen models`
- See the generated `CopyOfBuilder` class in `Blog.java`

**Expected behavior**
CLI should generate the code without the `super;` line.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac
 - Node Version: v12.16.1

**Comment by UnleashedMind** [comment]
reproduced, we are working on a fix.

**Comment by akshbhu** [comment]
Thanks for reporting this.

**Comment by azisoli** [comment]
we still have this problem we generated yesterday. what we should do ?

**Comment by akshbhu** [comment]
Hi @azisoli 

This is released in amplify latest version (4.27.3) today. 
Can you check with this and let me know if the issue still there?
